ignite-worker/src/main/java/org/opennms/poc/ignite/worker/ignite/service/WorkflowService.java

```diff
@@ -55,6 +64,11 @@ public void execute() {
         log.info("########### registered detector count {}", OsgiServiceHolder.getRegisteredDetectorCount());
         Optional<ServiceDetector> detector = OsgiServiceHolder.getDetector(workflow.getType());
         log.info("Detector is {}", detector.isPresent() ? detector.toString():"NOT FOUND");
+        Optional<ServiceMonitor> serviceMonitor = OsgiServiceHolder.getMonitor("blah");
+
+        for (int i=0;i< 5;i++) {
+            serviceMonitor.get().poll(null ,null).whenComplete((serviceMonitorResponse, exception ) -> log.info("got a response") );
```

Comment:
@artnaseef this good for now, or should do we something different here?

Reply:
@j-white ah, I think this is touching on a topic I neglected to raise.  The `load-em-up` REST endpoint uses different Ignite services than the Workflow driven by Twin updates.  Do we want to merge them, or perhaps drop the REST endpoint and this class altogether?  Responsibility for this mess lies with me.

Reply:
This class is deprecated now anyway.